### PR TITLE
Remove bootstrap from UI components

### PR DESCRIPTION
**Pull request by cipriandraghici93**

### CODE REVIEW

File: src/UI/Denominate/index.tsx [hunk 1]
```diff
@@ -63,12 +65,16 @@ const denominateValid = (props: DenominateType, erdLabel: string) => {
...
-      <span className='int-amount'>{valueParts[0]}</span>
+      <span className={styles.intAmount}>{valueParts[0]}</span>
       {valueParts.length > 1 && (
-        <span className='decimals'>.{valueParts[1]}</span>
+        <span className={styles.decimals}>.{valueParts[1]}</span>
       )}
       {showLabel && (
-        <span className={`symbol ${props.token ? 'text-muted' : ''}`}>
+        <span
+          className={`${styles.symbol} ${
```
Comment (arhtudormorar) [inline src/UI/Denominate/index.tsx:74]: classnames?

File: src/UI/TokenDetails/TokenSymbol/index.tsx [hunk 2]
```diff
@@ -36,18 +38,23 @@ export const Pool = ({
...
     </div>
   </div>
 );
 
 export const Locked = ({ children }: { children: React.ReactNode }) => (
-  <div className='token-symbol-locked'>
-    <div className='locked rounded-circle shadow'>
-      <div className='locked-icon'>
+  <div className={styles.tokenSymbolLocked}>
+    {/*<div className='locked rounded-circle shadow'>*/}
```
Comment (arhtudormorar) [inline src/UI/TokenDetails/TokenSymbol/index.tsx:53]: maybe remove comment?

File: src/UI/TokenDetails/TokenSymbol/token-symbol.scss [hunk 3]
```diff
@@ -154,3 +156,23 @@
     //box-shadow: 0 0 0 6px $card-inner-bg-light;
```
Comment (arhtudormorar) [inline src/UI/TokenDetails/TokenSymbol/token-symbol.scss:156]: commented code

File: src/UI/TransactionData/index.tsx [hunk 4]
```diff
@@ -34,7 +35,7 @@ const TransactionData = ({
         const [rest] = data.split(highlight);
         output = (
           <React.Fragment>
```
Comment (arhtudormorar) [inline src/UI/TransactionData/index.tsx:37]: dts package allows use of` <> `fragment

File: src/UI/TransactionData/index.tsx [hunk 5]
```diff
@@ -46,9 +47,9 @@ const TransactionData = ({
 
         output = (
           <React.Fragment>
```
Comment (arhtudormorar) [inline src/UI/TransactionData/index.tsx:49]: <>

File: src/UI/UsdValue/index.tsx [hunk 6]
```diff
@@ -9,7 +10,10 @@ const UsdValue = (props: {
   const { amount, usd, ...dataTestId } = props;
   const value = `≈ $${getUsdValue({ amount, usd })}`;
   return (
-    <small className='form-text text-secondary mt-0' {...dataTestId}>
+    <small
+      className={`${globalStyles.formText} ${globalStyles.textSecondary} ${globalStyles.mt0}`}
+      {...dataTestId}
```
Comment (arhtudormorar) [inline src/UI/UsdValue/index.tsx:15]: is this correct?

File: src/UI/ledger/LoginButton/index.tsx [hunk 7]
```diff
@@ -26,10 +28,12 @@ const LedgerLoginButton: (props: LedgerLoginButtonPropsType) => JSX.Element = ({
     className,
     shouldRenderDefaultCss,
     {
-      wrapper: `btn btn-primary px-sm-4 m-1 mx-sm-3 ${
+      wrapper: `${globalStyles.btn} ${globalStyles.btnPrimary} ${
+        globalStyles.px4
+      } ${globalStyles.m1} ${globalStyles.mx3} ${
```
Comment (arhtudormorar) [inline src/UI/ledger/LoginButton/index.tsx:33]: class conditionals classnames

File: src/UI/ledger/LoginModal/LedgerConnect.tsx [hunk 8]
```diff
@@ -11,29 +13,36 @@ const LedgerConnect = ({
...
-              <p className='text-danger d-flex justify-content-center align-items-center'>
+              <p
+                className={`${globalStyles.textDanger} ${globalStyles.flexRow} ${globalStyles.justifyContentCenter} ${globalStyles.alignItemsCenter}`}
+              >
                 {error}
               </p>
             )}
             <button
-              className='btn btn-primary px-4'
+              // className='btn btn-primary px-4'
```
Comment (arhtudormorar) [inline src/UI/ledger/LoginModal/LedgerConnect.tsx:44]: comment

File: src/UI/webWallet/LoginButton/index.tsx [hunk 9]
```diff
@@ -19,10 +20,12 @@ const WebWalletLoginButton: (
     token
   });
   const classes = getGeneratedClasses(className, shouldRenderDefaultCss, {
-    wrapper: `btn btn-primary px-sm-4 m-1 mx-sm-3 ${
+    wrapper: `${globalStyles.btn} ${globalStyles.btnPrimary} ${
+      globalStyles.px4
+    } ${globalStyles.m1} ${globalStyles.mx3} ${
```
Comment (arhtudormorar) [inline src/UI/webWallet/LoginButton/index.tsx:25]: classnames conditional

File: src/assets/sass/main.scss [hunk 10]
```diff
@@ -1,24 +1,15 @@
-$headings-font-weight: 300 !important;
-$primary: #1b46c2;
-$light: #fafafa;
-$a: #1392ff;
 
-
-@import './../../../node_modules/bootstrap/scss/bootstrap';
+//@import './../../../node_modules/bootstrap/scss/bootstrap';
```
Comment (arhtudormorar) [inline src/assets/sass/main.scss:2]: comment

File: src/assets/sass/main.scss [hunk 11]
```diff
@@ -1,24 +1,15 @@
...
 
 @import './themes/theme.scss';
 
 // Variables override
 @import './variables/variables';
 
 
-    @import './../../../node_modules/bootstrap/scss/reboot';
+//@import './../../../node_modules/bootstrap/scss/reboot';
```
Comment (arhtudormorar) [inline src/assets/sass/main.scss:10]: comment

File: src/assets/sass/modules/data/data.scss [hunk 12]
```diff
@@ -34,7 +34,7 @@
   .small-font {
     font-size: $font-size;
   }
-  @include media-breakpoint-up(sm) {
-    max-width: 39rem;
-  }
+  //@include media-breakpoint-up(sm) {
```
Comment (arhtudormorar) [inline src/assets/sass/modules/data/data.scss:37]: comment

File: src/assets/sass/modules/toast-messages/_toast-messages.scss [hunk 13]
```diff
@@ -5,12 +5,12 @@
   right: 1rem;
   z-index: 9999;
 
-  @include media-breakpoint-up(md) {
-    right: 2rem;
-    bottom: 2rem;
-    width: 22.5rem;
-    left: auto;
-  }
+  //@include media-breakpoint-up(md) {
```
Comment (arhtudormorar) [inline src/assets/sass/modules/toast-messages/_toast-messages.scss:8]: comment

File: src/assets/sass/modules/toast-messages/_toast-messages.scss [hunk 14]
```diff
@@ -45,18 +45,18 @@
...
-      &:first-child {
-        display: block;
-        transform: translateY(150%);
-
-        &.toast-visible {
-          transform: translateY(0);
-        }
-      }
-    }
+    //@include media-breakpoint-down(sm) {
```
Comment (arhtudormorar) [inline src/assets/sass/modules/toast-messages/_toast-messages.scss:48]: comment

File: src/assets/sass/themes/button.scss [hunk 15]
```diff
@@ -0,0 +1,104 @@
...
+  display: inline-block;
+  font-weight: 400;
+  color: #212529;
+  text-align: center;
+  vertical-align: middle;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  background-color: transparent;
```
Comment (arhtudormorar) [inline src/assets/sass/themes/button.scss:13]: better use of defined transparency (just an oppinion)

File: src/assets/sass/variables/_variables.scss [hunk 16]
```diff
@@ -1,6 +1,37 @@
-@import './../../../../node_modules/bootstrap/scss/functions';
-@import './../../../../node_modules/bootstrap/scss/variables';
-@import './../../../../node_modules/bootstrap/scss/mixins';
+//@import './../../../../node_modules/bootstrap/scss/functions';
```
Comment (arhtudormorar) [inline src/assets/sass/variables/_variables.scss:1]: comment

File: src/UI/UsdValue/index.tsx (hunk 6 above)
Comment (cipriandraghici93) [reply src/UI/UsdValue/index.tsx:15]: The logic was not changed. Only the `className` property was touched.

File: src/assets/sass/themes/button.scss (hunk 15 above)
Comment (cipriandraghici93) [reply src/assets/sass/themes/button.scss:13]: What is your suggestion?

File: src/UI/walletConnect/WalletConnectLoginButton/index.tsx [hunk 17]
```diff
@@ -28,10 +30,12 @@ const WalletConnectLoginButton = ({
     className,
     shouldRenderDefaultCss,
     {
-      wrapper: `btn btn-primary px-sm-4 m-1 mx-sm-3 ${
+      wrapper: `${globalStyles.btn} ${globalStyles.btnPrimary} ${
+        globalStyles.px4
+      } ${globalStyles.m1} ${globalStyles.mx3} ${
         buttonClassName != null ? buttonClassName : ''
```
Comment (StanislavSava) [inline src/UI/walletConnect/WalletConnectLoginButton/index.tsx:36]: let's use classnames for this

**Review by MiroMargineanu — APPROVED** [review]

**Pull request merged**